lib/ash_authentication/add_ons/confirmation.ex

```diff
@@ -0,0 +1,140 @@
+defmodule AshAuthentication.AddOn.Confirmation do
+  import AshAuthentication.Dsl
+
+  @moduledoc """
+  Confirmation support.
+
+  Sometimes when creating a new user, or changing a sensitive attribute (such as
+  their email address) you may want to for the user to confirm by way of sending
```

Comment:
I think there's a typo here???
```suggestion
  their email address) you may want wait to for the user to confirm by way of sending
```

Reply:
Nice catch!